scripts/get-binary-names-from-cargo-metadata.mjs

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env node
```

Comment:
does this mean we can get rid of the node dependency?

Reply:
ah, i finally made it to the bottom of your description -- is that something that we remove in another PR?

Reply:
The `node` dependency was implicit, so it's gone once we stop using it (as done here).